sdk/python/feast/infra/offline_stores/contrib/trino_offline_store/__init__.py

```diff
@@ -0,0 +1,32 @@
+import sys
+
+from .connectors.upload import upload_pandas_dataframe_to_trino
+from .connectors.utils import (
+    format_datetime,
+    format_pandas_row,
+    pandas_dataframe_fix_batches,
+    pyarrow_schema_from_dataframe,
+    trino_table_schema_from_dataframe,
+)
+from .trino import TrinoOfflineStore, TrinoOfflineStoreConfig
+from .trino_source import TrinoOptions, TrinoSource
+
+if sys.version_info >= (3, 8):
```

Comment:
This logic should actually be cleaned. There is not need to support multiple versions as the plug-in will now be part of the main repo

Reply:
So let’s clean the file as if it will always be used by Feast 0.1.9+